### ISSUE TITLE
Using SSH with 1Password SSH agent

### ISSUE THREAD

**Issue by suasgn** [TablePlus/TablePlus-Windows#569]
### Please answer the questions below, it helps us to track the issue.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgresSQL, MySQL, MariaDB

3. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
build 570 (macos)

5. The steps to reproduce this issue:
No specific step

Noted: If the bug is related to data, please attach an example SQL data.

---

I have check the compatibility list on the https://developer.1password.com/docs/ssh/agent/compatibility/, and it seems TablePlus is still doesn't support the SSH Agent.

TablePlus is allowed us to access the database via SSH tunnel, but it is limited only by using the private key.

Exporting the private key to disk is against the purpose and benefits of 1Password and other password managers offered.

Quoted from the 1Password site about the feature:

> The 1Password SSH agent uses the SSH keys you have saved in 1Password to seamlessly integrate with your Git and SSH workflows. It authenticates your Git and SSH clients without those clients ever being able to read your private key.
> 
> In fact, your private key never even leaves the 1Password app. The SSH agent works with the SSH keys stored in your Personal or Private vault, but never without your consent. Only SSH clients you explicitly authorize will be able to use your SSH keys until 1Password locks.

**Comment by suasgn** [comment]
related issue #495

**Comment by belaarany** [comment]
Any progress on this?

**Comment by rctneil** [comment]
I'd love to know about this.  Just spent an hour trying to get it to work usig the keys in 1PW, only to find I need to export the key into .ssh and use that for the connection.

**Comment by flemming-pr** [comment]
Any update on this? It would be nice to not have any key stored on the drive.

**Comment by pkeogan** [comment]
bump, looking to do the same. 1password + TablePlus on windows 11.

**Comment by hannesw** [comment]
+1 on this feature

**Comment by huyphams** [comment]
sorry, close wrong issue

**Comment by suasgn** [comment]
> sorry, close wrong issue

this mean there is a chance for 1Password?

**Comment by huyphams** [comment]
Yes, @suasgn, and maybe soon, once it works on macOS. But I think it will work with the existing build with the correct configuration. However, I find it easier to add it natively in TablePlus.
